src/update.js

```diff
@@ -0,0 +1,12 @@
+export function checked(element, arr) {
+  const num = element.id;
+  arr[num].completed = true;
+  localStorage.setItem('Do-Lists', JSON.stringify(arr));
+  window.location.reload();
+}
```

Comment:
- [ ] Please kindly work on the checked functionality as it only checks and does not uncheck.